README.md

```diff
@@ -30,14 +30,17 @@ This currently includes all cru.org styling from cru.scss.
 
 #### Branded checkout basic example
 
-Add the following code to your page where appropriate:
+Add the following code to your page where appropriate. You must change the value of the `code` and `tsys-env` attributes:
+- `code` must be changed to the designation number you want users to give to.
+- `tsys-env` must be changed to match your environment. It is provided by DPS when adding your domain to the TSYS whitelist.
```

Comment:
I was actually wrong about this value being specific to the environment.
The current values are 'cru' and 'familyLife'.

Perhaps call this 'tsys-device'? Or 'tsys-device-id'?